diff --git a/tests/phpunit/includes/bootstrap.php b/tests/phpunit/includes/bootstrap.php
index 23faca64e5..b316cd694b 100644
--- a/tests/phpunit/includes/bootstrap.php
+++ b/tests/phpunit/includes/bootstrap.php
@@ -6,9 +6,9 @@
 /**
  * Compatibility with PHPUnit 6+
  */
-if ( class_exists( 'PHPUnit\Runner\Version' ) ) {
-	require_once dirname( __FILE__ ) . '/phpunit6/compat.php';
-}
+//if ( class_exists( 'PHPUnit\Runner\Version' ) ) {
+//	require_once dirname( __FILE__ ) . '/phpunit6/compat.php';
+//}
 
 if ( defined( 'WP_TESTS_CONFIG_FILE_PATH' ) ) {
 	$config_file_path = WP_TESTS_CONFIG_FILE_PATH;
